checkST2110.js

```diff
@@ -70,68 +72,117 @@ a=ts-refclk:ptp=IEEE1588-2008:39-A7-94-FF-FE-07-CB-D0:37
 a=mediaclk:direct=0
 a=mid:secondary`;
 
+
+
+const specExample22 = `v=0 
```

Comment:
I think it would be nice to start putting the source for examples, e.g., if they come from a particular spec. What do you think?